viewer/packages/cad-geometry-loaders/src/sector/culling/TakenSectorMap.ts

```diff
@@ -0,0 +1,117 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import { TakenSectorTree } from './TakenSectorTree';
+import {
+  PrioritizedWantedSector,
+  DetermineSectorCostDelegate,
+  SectorCost,
+  addSectorCost,
+  SectorLoadingSpent
+} from './types';
+import { CadModelSectorBudget } from '../../CadModelSectorBudget';
+
+import { CadModelMetadata, LevelOfDetail } from '@reveal/cad-parsers';
+
+import assert from 'assert';
+
+export class TakenSectorMap {
```

Comment:
Pretty sure this file has been moved/renamed and that this class now is unused. Verify and delete this.